core/src/net/mod.rs

```diff
@@ -1452,6 +1462,12 @@ pub mod net_test_helpers {
                 NetworkStatus::CriticalNetworkFailure => {
                     self.critical_network_failure += 1;
                 }
+                NetworkStatus::BlockedIpNet(_) => {}
+                NetworkStatus::AllowedIpNet(_) => {}
+                NetworkStatus::AllowedIp(_) => {}
+                NetworkStatus::BlockedSystem(_) => {}
+                NetworkStatus::BlockedIp(_) => {}
+                NetworkStatus::AllowedSystem(_) => {}
```

Comment:
Use `()` instead of `{}`

Reply:
How about 😄 
```rust
NetworkStatus::CriticalNetworkFailure => {
    self.critical_network_failure += 1;
}
// Ignore others
_ => ()
```

Reply:
I changed it to () instead.

I prefer to keep it explicit which cases are ignored rather than the "ignore others" approach. Makes it easier to understand while reading it since the code now also communicates explicitly what it doesn't do, which cases it doesn't handle.